.github/workflows/maven-publish.yml

```diff
@@ -35,4 +35,4 @@ jobs:
           export SONATYPE_PASSWORD=$(aws secretsmanager get-secret-value --secret-id maven-snapshots-password --query SecretString --output text)
           echo "::add-mask::$SONATYPE_USERNAME"
           echo "::add-mask::$SONATYPE_PASSWORD"
-          ./gradlew publishPluginZipPublicationToSnapshotsRepository
+          ./gradlew publishPluginZipPublicationToSnapshotsRepository publishJarsPublicationToSnapshotsRepository
```

Comment:
Is this new gradle task "publishJarsPublicationToSnapshotsRepository" introduced by a newer version of gradle? I don't see this task in my local env.

Reply:
Perhaps, you have to rebuild the project to refresh the caches